app/models/manageiq/providers/amazon/cloud_manager/event_parser.rb

```diff
@@ -53,13 +53,13 @@ def self.event_to_hash(event, ems_id)
     end
 
     log_header = "ems_id: [#{ems_id}] " unless ems_id.nil?
-    _log.debug("#{log_header}event: [#{event[:message]}]")
+    _log.debug("#{log_header}event: [#{event_hash[:message]}]")
```

Comment:
Unrelated to this change, but this was always `nil` because we set `:message` in the `event_hash` it isn't in the native `event` payload from amazon.  Think this was just a mistake in the original implementation 609f666a